src/msg/async/rdma/RDMAServerSocketImpl.cc

```diff
@@ -97,7 +97,7 @@ int RDMAServerSocketImpl::accept(ConnectedSocket *sock, const SocketOptions &opt
     ::close(sd);
     return -errno;
   }
-  net.set_priority(sd, opt.priority);
+  net.set_priority(sd, opt.priority, out->get_family());
```

Comment:
Hi @yuyuyu101, @robbat2 , 
In the original commit: "msg/async: support IPv6 QoS. #13418" (https://github.com/ceph/ceph/pull/13418), the call to "set_priority()" is done without check if "out" is NULL. We need to first check if out is not NULL before sending it to "set_priority()". What should be default value if "out" is NULL?

For convenience I copied and marked the essential code to see the problem.

In PosixStack.cc::
260        int PosixServerSocketImpl::accept(ConnectedSocket *sock, const SocketOptions &opt, entity_addr_t *out, Worker *w) {
                …
281		handler.set_priority(sd, opt.priority, out->get_family()); 
…  
285		if (out)
286		     out->set_sockaddr((sockaddr*)&ss);
287		return 0;
288       } 

Thanks

@Adirl @orendu